samples/host_verify/CMakeLists.txt

```diff
@@ -1,12 +1,14 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
+set(COMPONENT OEHOSTVERIFY)
+
 cmake_minimum_required(VERSION 3.11)
 
 project("Remote Host-side Enclave Verification Sample" LANGUAGES C CXX)
 
 if (NOT BUILD_HOSTVERIFY_INLINE)
-  find_package(OpenEnclave CONFIG REQUIRED)
+  find_package(OpenEnclave COMPONENTS host_verify)
```

Comment:
I was trying to see if it would locate the host_verify component only, but it did not seem to be the case, so I will revert it to the original version. Thank you for all the other details! I will update it again just to clean up.

Reply:
This change seem still there? Can you double check?